app/helpers/application_helper.rb

```diff
@@ -128,8 +128,9 @@ def link_to_function_if_authorized(name, function, options = {}, html_options =
 
   def display_delete_if_authorized(options = {}, html_options = {})
     text = options.delete(:text) || _("Delete")
+    method = options.delete(:method) || :delete
     options = {:auth_action => :destroy}.merge(options)
-    html_options = { :data => { :confirm => _('Are you sure?') }, :method => :delete, :class => 'delete' }.merge(html_options)
+    html_options = { :data => { :confirm => _('Are you sure?') }, :method => method, :class => 'delete' }.merge(html_options)
```

Comment:
Yes, for the `recreate` action, which I also ask "Are you sure?"
Regardless, the whole "Are you sure?" scenario should be rewritten, imo :smile: